CONTRIBUTING.md

```diff
@@ -125,7 +125,7 @@ If you are adding a new file it should have a header like below.  The easiest
 way to add such header is to run `make fmt`.
 
 ```
-// Copyright (c) 2017 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
these changes don't belong to this PR, please sync the master and rebase your branch off of it